pycgmes/utils/base.py

```diff
@@ -129,14 +47,21 @@ def to_dict(self) -> dict[str, "CgmesAttributeTypes"]:
 
         """
         attrs = {f.name: getattr(self, f.name) for f in fields(self)}
-        attrs["__class__"] = self.resource_name
+        attrs["__class__"] = self.apparent_name()
         return attrs
 
     @cached_property
     def resource_name(self) -> str:
         """Returns the resource type."""
         return self.__class__.__name__
 
+    @cached_property
+    def namespace(self) -> str:
```

Comment:
so this is inherited, and unless we overwrite it a point of the inheritance chain, it stays "cim"

Reply:
Indeed, all default attributes are in the 'cim' namespace, so it makes sense to keep it as cim by default, with the option to override it for custom attributes.